.gitignore

```diff
@@ -0,0 +1,3 @@
+.htaccess
+debug.log
+README.md
```

Comment:
Every file needs to have an empty line at then, this is to ensure compatibility with other file systems.